task_2_nbu/Extractor.py

```diff
@@ -0,0 +1,44 @@
+import time
+
+import requests
+
+Currency = ['USD', 'EUR', 'MXN', 'INR', 'IRR', 'GEL']
+
+
+def check_month(month: int) -> str:
+    """check month, if < 10 :return like 0month"""
+    return f'0{month}' if month < 10 else month
+
+
+def check_day(day: int) -> str:
+    """check day, if < 10 :return like 0day"""
+    return f'0{day}' if day < 10 else day
```

Comment:
1. it is better to reuse the code;
2. in this special case it would be great to use string formatting for leading zero integers https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting